src/navigation/Nav.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Nav from './Nav';
+
+const renderComponent = props => shallow(<Nav {...props} />);
+
+describe('Nav', () => {
+	const navSwarm = renderComponent({
+		media: { isAtMediumUp: true },
+		self: { status: 'active', name: 'Jeff Cleft' },
+	});
+	const navScript = renderComponent({
+		media: { isAtLargeUp: true },
+		self: { status: 'prereg' },
+	});
```

Comment:
i feel like there are other cases besides the 2 above that should be unit tested?